.prettierignore

```diff
@@ -16,3 +12,17 @@ __benchmarks_results__/
 # HTML files are used by tests and may include deliberately invalid/ugly HTML
 # e.g. <input> without self-closing tag, extra leading/trailing whitespace, etc.
 *.html
+
+# Auto-generated fixture outputs
+**/fixtures/**/expected.*
+**/fixtures/**/error.*
+packages/@lwc/template-compiler/src/__tests__/fixtures/**/ast.json
+packages/@lwc/template-compiler/src/__tests__/fixtures/**/metadata.json
+
+# Fixture inputs potentially containing deliberate errors
+packages/@lwc/style-compiler/src/__tests__/fixtures/**/actual.css
+packages/@lwc/babel-plugin-component/src/__tests__/fixtures/**/actual.js
+
+# TODO: Why do these have JS in them?
```

Comment:
These have JS in them because we are testing how Node/module resolution works for `.js` files with a particular directory structure / name.